auth-lib/src/store/java/com/spotify/sdk/android/auth/store/PlayStoreHandler.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2015-2016 Spotify AB
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.sdk.android.auth.store;
+
+import android.app.Activity;
+import android.content.ComponentName;
+import android.content.Intent;
+import android.net.Uri;
+import android.util.Log;
+
+import com.spotify.sdk.android.auth.AuthorizationHandler;
+import com.spotify.sdk.android.auth.AuthorizationRequest;
+
+/**
+ * An AuthorizationHandler that opens the play store to download the main spotify application
```

Comment:
Nit: since it's public documentation - Spotify with capital "S"

Reply:
I was also wondering if you could add a similar description to the BrowserHandler because it doesn't have one. "An AuthorizationHandler that loads the auth page inside a Custom Tab" would be fine 🙂

Reply:
That's a good habit to take, will capitalize it.
Sure thing.